readme.md

```diff
@@ -1,70 +1,100 @@
+2. Install all of the project dependencies using npm:
 
-### Run!
+    ```sh
+    npm install
+    ```
 
-Ok, time to run the app. Npm takes care of business again:
+3. Once installed you can run the Webpack development server:
 
-`$ npm start`
+    ```sh
+    npm start
+    ```
 
-This will fire up the wepback dev server. It's not important to understand how it works for now.
+    Now open [http://localhost:8080/](http://localhost:8080/) in your browser and you should see a basic HTML page and if you open the developer console in the browser you should see a welcome message which confirms everything is loading and running correctly.
 
-Open [http://localhost:8080/](http://localhost:8080/) in your browser and you should see a basic html page. If you open the development console in your browser you should see a welcome message.
+    You will need to press `ctrl + c` to stop the server.
 
-### Development
+4. And to run your tests use the following command:
 
-The source files are located in `src/`. If you change a file, webpack will reload your files and refresh the browser for you.
+    ```sh
+    npm test
+    ```
 
-### Test
+## The exercise
 
-You can run the example tests:
+### Goal
 
-`$ npm test`
+We would like you to create a client-side web application to help a child understand multiplication. Your application will display a grid of numbers, when a child selects a number the multiples of that chosen number will be highlighted.
 
-## The exercise
+### Functional requirements
 
-__Goal__: Production of a client-side Web application to help a child understand multiplication.
+- The application will display numbers from 1 to 144.
+- When a number is clicked all numbers which are a multiple of the chosen number must be highlighted.
+- When a new number is chosen the previously highlighted numbers must be updated.
 
-__Duration__: This test is intended to take ~2 hours but may take more
+### Visual requirements
 
-__Description__: The application will display a grid of all the numbers from 1 to 144 evenly laid out with a large clickable box around each number. The number should be centered in each box. When the child clicks a box then all the numbers which are a multiple of that number should be highlighted in some way. Clicking on a second number removes the highlighting of the previous selection, then highlights the multiples of the newly selected number.
+- Each number must be within a large clickable box.
+- Each number must be horizontally and vertically centered within their clickable box.
+- The boxes must be laid out evenly within a grid.
+- On mobile and small screen devices 1 number must be displayed in each grid row.
+- On tablet and middle sized devices 2 numbers must be displayed in each grid row.
+- On desktop and large screen devices 3 numbers must be displayed in each grid row.
+- The grid should have a maximum width so that the clickable boxes do not get too large.
+- The grid should be horizontally centered on the screen.
```

Comment:
```suggestion
- Each number must be within a large clickable box.
- Each number must be horizontally and vertically centered within their clickable box.
- The boxes must be laid out evenly within a grid.
- On mobile and small screen devices, one number must be displayed in each grid row.
- On tablet and middle sized devices, two numbers must be displayed in each grid row.
- On desktop and large screen devices, three numbers must be displayed in each grid row.
- The grid should have a maximum width so that the clickable boxes do not get too large.
- The grid should be horizontally centered on the screen.
```